internal/test/test.go

```diff
@@ -26,7 +27,8 @@ import (
 func CmdValidator(t *testing.T, subject *cobra.Command, nSubCommands int, flags []string) {
 	t.Helper()
 	a := assert.New(t)
-	a.Len(subject.Commands(), nSubCommands)
+	assertMsg := fmt.Sprintf("Sub command count mismatch. Expected %d, got %d. Check the CmdValidator for your command.", len(subject.Commands()), nSubCommands)
```

Comment:
@gssbzn What we discussed earlier, adding extra help text when these tests fail, for future us